ggml-cuda.cu

```diff
@@ -255,6 +297,23 @@ static void dequantize_row_q8_0_cuda(const void * vx, float * y, int k, cudaStre
     dequantize_block_q8_0<<<nb, 1, 0, stream>>>(vx, y);
 }
 
+static void dequantize_mul_mat_q4_0_cuda(const void * vx, const float * y, float * dst, const int ncols, const int nrows, cudaStream_t stream) {
```

Comment:
This should probably be renamed to `dequantize_mul_mat_vec_q4_0_cuda()` or something similar? I didn't read the PR description very carefully at first, and spent some time scratching my head and wondering if there is a missing dimension for `y` in the kernel.

Reply:
I agree that something along the lines of `mat_mul_vec` would be a better name; I just forgot to change it.